e2e-tests/portal/cypress/integration/journeys/maker/loan/loan-guarantee-details.spec.js

```diff
@@ -154,6 +154,49 @@ context('Loan Guarantee Details', () => {
         });
       });
     });
+
+    it('should show validation errors when incorrect date inputs and show invalid date on contract page', () => {
+      goToPage(deal);
```

Comment:
Please specify a test case with blank input and input with spaces.
I know validation will fail but this is to ensure we have good input coverage.

Reply:
done